doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -506,6 +506,15 @@ The issues in this section are related to the :ref:`ug_matter` protocol.
 
 .. rst-class:: v2-9-0
 
+NCSDK-28567: Swap time after DFU takes a very long time
```

Comment:
seems like the issue affects 2.8.0 as well

Reply:
Right, thanks